connector/alertsgenconnector/ingest.go

```diff
@@ -0,0 +1,697 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package alertsgenconnector // import "github.com/open-telemetry/opentelemetry-collector-contrib/connector/alertsgenconnector"
```

Comment:
this file should move under internal, and probably break in 2 between all the buffer impls and the logic.